guides/common/modules/proc_running-ansible-roles-during-host-registration.adoc

```diff
@@ -0,0 +1,18 @@
+[id="running-ansible-roles-during-host-registration"]
+= Running Ansible roles during host registration
+
+You can run Ansible roles when you are registering a host to {Project}.
+
+.Prerequisites
+ifndef::satellite[]
+* Ansible integration is enabled on {Project}.
+For more information, see {ManagingConfigurationsAnsibleDocURL}Enabling_Ansible_Integration_with_{project-context}_ansible[Enabling Ansible integration with {Project}] in _{ManagingConfigurationsAnsibleDocTitle}_.
+endif::[]
+* The required Ansible roles have been imported to {ProjectServer}.
```

Comment:
```suggestion
* The required Ansible roles have been imported from your {SmartProxy} to {Project}.
```

If you register a host to Smart Proxy Server, then you'd need to import the Ansible roles from your Smart Proxy server into Foreman.

See Configure > Ansible > Roles; then click import and you will see a list of Smart Proxies. I assume that you need to import them from the proper Smart Proxy that the host is registered to.